atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleKeyValueServiceConfig.java

```diff
@@ -30,6 +30,16 @@
 
     public abstract JdbcHandler jdbcHandler();
 
+    @Value.Default
+    public String singleOverflowTable() {
+        return "atlas_overflow";
+    }
+
+    @Value.Default
+    public String overflowTablePrefix() {
+        return "ao_";
```

Comment:
Will this ever be used in OSS atlas, or will it just always be overwritten by the internal product using this config.  If so, I don't think it should have a default.